yb-voyager/cmd/fallBackSynchronize.go

```diff
@@ -53,3 +59,25 @@ func init() {
 func hideExportFlagsInFallBackCmds(cmd *cobra.Command) {
 	cmd.Flags().Lookup("parallel-jobs").Hidden = true
 }
+
+func initSourceConfFromTargetConf() error {
+	msr, err := metaDB.GetMigrationStatusRecord()
+	if err != nil {
+		return fmt.Errorf("get migration status record: %v", err)
+	}
+	targetConf := msr.TargetDBConf
```

Comment:
[REFACTORING, optional comment] How about moving all the connection related fields in a separate struct `DBConnParams` and embedding it in both source and target conf? That will allow significant code reuse. Here also, you could then simply do `source.DBConnParams = targetConf.DBConnParams`

Reply:
Great idea! Hopefully vscode facilitates such a refactor easily? 
Will take this up as a separate task, as want to merge this asap to enable testing.. Created a JIRA for 1.7 - https://yugabyte.atlassian.net/browse/DB-8345